deploy/manifests/dev/us-east-2/lassie-event-recorder/patch.yaml

```diff
@@ -20,6 +20,11 @@ spec:
                 secretKeyRef:
                   name: lassie-event-recorder-db-creds
                   key: dsn
+            - name: LASSIE_EVENT_RECORDER_MONGO
+              valueFrom:
+                secretKeyRef:
+                  name: lassie-event-recorder-mongo-creds
+                  key: mongoaddr
```

Comment:
I think this is 'mongo'
https://github.com/filecoin-project/lassie-event-recorder/blob/main/cmd/recorder/main.go#L26